src/components/rating/rating.tsx

```diff
@@ -3,27 +3,38 @@ import {
   Element,
   Event,
   EventEmitter,
-  Fragment,
+  FunctionalComponent,
   h,
-  Listen,
+  Host,
   Method,
   Prop,
   State,
-  VNode
+  Watch
 } from "@stencil/core";
 import { guid } from "../../utils/guid";
 import { Scale } from "../interfaces";
 import { LabelableComponent, connectLabel, disconnectLabel } from "../../utils/label";
 import { connectForm, disconnectForm, FormComponent, HiddenFormInputSlot } from "../../utils/form";
 import { TEXT } from "./resources";
 import { InteractiveComponent, updateHostInteraction } from "../../utils/interactive";
-import { isActivationKey } from "../../utils/key";
 import {
   setUpLoadableComponent,
   setComponentLoaded,
   LoadableComponent,
   componentLoaded
 } from "../../utils/loadable";
+import { Star, StarIconProps } from "./interfaces";
+
+const StarIcon: FunctionalComponent<StarIconProps> = ({ icon, scale, partial }) => (
+  <calcite-icon
+    {...{
+      "aria-hidden": "true",
```

Comment:
"aria-hidden" isn't needed here as `calcite-icon` already sets it to true.

Reply:
sorry I made you mention this twice. thanks for the catch